.gitignore

```diff
@@ -31,6 +31,7 @@
 .env.test.local
 .env.production.local
 .env
+.nvmrc
```

Comment:
can we remove the changes in this file as it might be different for each user depending on their OS and setup